lambdas/backend/_common/customers-controller.js

```diff
@@ -323,16 +319,6 @@ function updateCustomerApiKeyId(cognitoIdentityId, apiKeyId, error, success) {
     })
 }
 
-// function getUsagePlans(error, callback) {
-//     const params = {
-//         limit: 1000
-//     }
-//     apigateway.getUsagePlans(params, (err, data) => {
-//         if (err) error(err)
-//         else callback(data)
-//     })
-// }
-
```

Comment:
Deleting this snippet because it's clearly not being used, and is obsoleted by this PR.

Reply:
+1